### PR TITLE
Add FAQ entry on using certificates when connecting to Serverless.

### PR DESCRIPTION
**Pull request by ianjevans**

### CODE REVIEW

File: cockroachcloud/serverless-faqs.md [hunk 1]
```diff
@@ -105,6 +105,14 @@ You can submit feedback or log any bugs you find through [this survey](https://f
...
+
+Some drivers and ORMs don't use the system root certificates, however. In those cases, you need to download a root certificate file and configure your client to use that certificate when connecting to your cluster. You can [download the certificate](connect-to-a-serverless-cluster.html#step-2-connect-to-your-cluster) by following the instructions in the {{ site.data.products.db }} Console. Configure your client to use this certificate (for example, by setting `sslrootcert=<path to the root certificate>` in your connection string) and to use SSL (for example, by setting `sslmode=verify-full` in your connection string) to connect to your cluster.
```
Comment (stbof) [inline cockroachcloud/serverless-faqs.md:112]:
```suggestion
However, some drivers and ORMs don't use the system root certificates. In those cases, you need to download a root certificate file and configure your client to use that certificate when connecting to your cluster. You can [download the certificate](connect-to-a-serverless-cluster.html#step-2-connect-to-your-cluster) by following the instructions in the {{ site.data.products.db }} Console. Configure your client to use this certificate (for example, by setting `sslrootcert=<path to the root certificate>` in your connection string) and to use SSL (for example, by setting `sslmode=verify-full` in your connection string) to connect to your cluster.
```

**Review by stbof — APPROVED** [review]
Minor wording suggestion.

File: cockroachcloud/serverless-faqs.md (hunk 1 above)
Comment (stbof) [reply cockroachcloud/serverless-faqs.md:112]: Are there other ways to configure the client?

Comment (ianjevans) [reply cockroachcloud/serverless-faqs.md:112]: There's no standard way, so it depends on the driver/ORM. Many use the libpq connection string format that we favor, but others (JDBC and C# prominently) use different connection strings or have individual parameters.

File: cockroachcloud/connect-to-a-serverless-cluster.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ To connect to your cluster with the [built-in SQL client](../{{site.versions["st
 
     {% include cockroachcloud/download-the-binary.md %}
 
-1. In your terminal, run the second command from the dialog to create a new `certs` directory on your local machine and download the CA certificate to that directory:
+1. In your terminal, run the second command from the dialog to download the CA certificate to the default PostgreSQL certificate directory:
```
Comment (gemma-shay) [inline cockroachcloud/connect-to-a-serverless-cluster.md:65]: Looking at the current **Connect** dialog, it seems like the second command was removed?

**Pull request merged**